src/Raven.Client/Documents/Commands/Batches/BatchOptions.cs

```diff
@@ -14,7 +16,7 @@ public sealed class BatchOptions
     public sealed class IndexBatchOptions
     {
         public bool WaitForIndexes { get; set; }
-        public TimeSpan WaitForIndexesTimeout { get; set; }
+        public TimeSpan? WaitForIndexesTimeout { get; set; }
```

Comment:
Why do we need this change?

Reply:
this shouldn't be reused, this is referring to 'batching' so... Session.SaveChanges (and the batch operation it is using) and we are in Patching context